Source/CMakeLists.txt

```diff
@@ -87,6 +87,7 @@ else()
     message(FATAL_ERROR "Invalid BUILD_TYPE: '${BUILD_TYPE}'")
 endif()
 
+set (HIDE_NON_EXTERNAL_SYMBOLS true)
```

Comment:
Think we need to remove this setting, but maybe good to, by default, already enable this in our builds so it aligns better with the latest Yocto builds and keeps the binaries smaller.